tests/Backend/Exasol/Workspace/WorkspaceLoadTest.php

```diff
@@ -2,12 +2,56 @@
 
 namespace Keboola\Test\Backend\Exasol\Workspace;
 
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Workspaces;
+use Keboola\Test\Backend\Workspaces\Backend\InputMappingConverter;
 use Keboola\Test\Backend\Workspaces\WorkspacesLoadTest;
 
 class WorkspaceLoadTest extends WorkspacesLoadTest
 {
     public function testDottedDestination()
     {
-        $this->markTestSkipped('Dotted tables not supported');
+        $workspaces = new Workspaces($this->workspaceSapiClient);
+        $workspace = $workspaces->createWorkspace();
+
+        // Create a table of sample data
+        $importFile = __DIR__ . '/../../../_data/languages.csv';
+        $tableId = $this->_client->createTable(
+            $this->getTestBucketId(self::STAGE_IN),
+            'languages_dotted',
+            new CsvFile($importFile)
+        );
+
+        $options = [
+            "input" => [
+                [
+                    "source" => $tableId,
+                    "destination" => "dotted.destination",
+                    "columns" => [
+                        [
+                            "source" => "id",
+                            "type" => "INTEGER",
+                        ],
+                    ],
+                ],
+            ],
+        ];
+
+        $options = InputMappingConverter::convertInputColumnsTypesForBackend(
+            $workspace['connection']['backend'],
+            $options
+        );
+
+        try {
+            $workspaces->loadWorkspaceData($workspace['id'], $options);
+            self::fail("Dotted destination is not supported in exasol");
+        } catch (ClientException $e) {
+            self::assertEquals(
+                "Invalid table name: Only alphanumeric characters dash and underscores are allowed.",
```

Comment:
Tady mi ten test neprochazi, pac mi to vraci jen hlasku:
`Only alphanumeric characters dash and underscores are allowed.`
Ta je definovana v legacy-app/library/Keboola/Validate/Exasol/ObjectNameValidate.php:18

Resp. neni tam ocekavane pole s `0` indexem:
```php
array (
  'regexNotMatch' => 'Only alphanumeric characters dash and underscores are allowed.',
)
```

Ale mozna mam neco spatne nastaveny...

Reply:
vyreseno